### DISCUSSION TITLE
HTTP와 HTTPS의 차이점을 설명해주세요

### DISCUSSION THREAD

**Discussion by sa02045** [prgrms-web-devcourse/We-are-the-MASTERS-OF-HTTP#74] [Interview Question]
`+`

- 네트워크 계층에서 어떤 차이점이 있는지
- HTTPS의 보안이 왜 더 좋은지
- HTTPS의 장점과 단점은 무엇인지
- HTTPS의 핸드세이크 과정은 어떻게 이루어지는지 [참고](https://aws-hyoh.tistory.com/entry/HTTPS-%ED%86%B5%EC%8B%A0%EA%B3%BC%EC%A0%95-%EC%89%BD%EA%B2%8C-%EC%9D%B4%ED%95%B4%ED%95%98%EA%B8%B0-3SSL-Handshake)

**Comment by Hyevvy** [comment]
HTTPS는 SSL (secure socket Layer) 인증서를 사용하는 HTTP이다. HTTP 프로토콜을 전송계층의 TCP위에서 동작하는데, HTTPS는 SSL이라는 보안계층이 전송계층 위에 올라간다. 또한 이 SSL 암호화 통신은 공개키 암호화 방식을 통해 구현된다.

HTTPS의 장점은 보안적인 측면뿐만 아니라 검색엔진 최적화(SEO)도 있다. 구글이 HTTPS 웹 사이트에 가산점을 주고, 유저들도 https 사이트가 더 안전하다고 느껴서 방문율을 높일 수 있다.
